src/components/accessories/feedback/type.ts

```diff
@@ -0,0 +1,7 @@
+export interface FeedbackParams {
```

Comment:
The pattern for type / interface is T (or I) + the Reference name (or OwnProps if it's relative to a single element).
So you should change the name or in IOwnProps or IFeedbackProps, make your choice :)

Reply:
It's OK !